Makefile

```diff
@@ -28,9 +28,7 @@ publish:
 VERSION := $(shell toml get Cargo.toml workspace.package.version --raw)
 
 docker-build:
-	docker build . -t mailtutan/mailtutan:$(VERSION)
+	docker buildx build --platform linux/amd64,linux/arm64 -t mailtutan/mailtutan:$(VERSION) .
 
 docker-push:
-	docker tag mailtutan/mailtutan:$(VERSION) mailtutan/mailtutan:latest
-	docker push mailtutan/mailtutan:$(VERSION)
-	docker push mailtutan/mailtutan:latest
+	docker buildx build --platform linux/amd64,linux/arm64 -t mailtutan/mailtutan:$(VERSION) -t mailtutan/mailtutan:latest --push .
```

Comment:
Make this long line multiple physical lines.

Reply:
Fixed in https://github.com/mailtutan/mailtutan/pull/73/commits/8e02894822ebc52320fe3095b3a9466f67e79b4c